docs/components/board/_index.md

```diff
@@ -1518,7 +1583,7 @@ interrupt.add_callback(callback_queue)
 
 **Parameters:**
 
-- `callback` [(chan Tick)](https://go.dev/tour/concurrency/2): The channel to add as a listener for when the state of the GPIO pin this interrupt is [configured for](#digital_interrupts) changes between high and low.
+- `callback` ([chan](https://go.dev/tour/concurrency/2) [Tick](https://pkg.go.dev/go.viam.com/rdk/components/board#Tick)): The channel to add as a listener for when the state of the GPIO pin this interrupt is [configured for](#digital_interrupts) changes between high and low.
```

Comment:
just FYI that `Close`, `AddCallback` and `Tick` are going to be removed from the interface in the next release. They are also not part of the API so not sure that they ever should have been documented here ?

Reply:
Oh interesting. Thanks for letting me know, I can make a pr to remove them. Are there any associated tickets I can track? 

I assume you mean `Close` for the digitalInterrupt type and not for board api? I think we documented them bc they were in the interface on Go (and w/ Python for close)-- sorry for the confusion, how would commands be implemented in the interface without API endpoints (how could we differentiate)?

Reply:
yeah Close for the digitalInterrupt, board will still have `Close`. Theres a ticket `here` that was split into multiple PRs so Close is done and others are in progress. 

Ideally the Go interfaces would have parity with the API but we have some components (might just be board) where that is not the case and there are methods on the interface that do not have APIs. Since theres no API they can't be called client side but they do still need to be implemented when you make a driver or module for that component. (so actually maybe these methods should be documented bc of that).

Reply:
thanks for the explanation, sounds good!